packages/x-tailwindcss/src/types.ts

```diff
@@ -90,20 +91,7 @@ export type DynamicCssStylesOptions = Dictionary<{
  *
  * @public
  */
-export type TailwindHelpers = {
-  theme: (themePath: string) => string;
-  addBase: (cssStyleOptions: CssStyleOptions) => void;
-  addComponents: (cssStyleOptions: CssStyleOptions) => void;
-  matchComponents: (
-    components: Dictionary<(value: any) => Dictionary>,
-    options: { values: any }
-  ) => void;
-  addUtilities: (cssStyleOptions: CssStyleOptions) => void;
-  matchUtilities: (
-    utilities: Dictionary<(value: any) => Dictionary>,
-    options: { values: any }
-  ) => void;
-};
+export type TailwindHelpers = PluginAPI;
```

Comment:
I know that preserving the alias results in less changes, but should we just use `PluginAPI` directly?

Reply:
Because it is not a "Public" type. Tailwind is not exporting it directly. So if they change this type or its location, with this alias, we only have to change in one place